be/src/exec/pipeline/runtime_filter_types.h

```diff
@@ -307,12 +305,17 @@ class PartialRuntimeFilterMerger {
             desc->set_runtime_filter(filter);
         }
 
-        for (auto& params : _partial_bloom_filter_build_params) {
+        for (auto& opt_params : _partial_bloom_filter_build_params) {
```

Comment:
If any partial bloom filter is null, do we could merge the bloom filter? in other words，Is the merged bloom filter is meaningful and useful?

Reply:
in three cases:
partial bloom filter is null have two different cases:
1. partial bf failed to be construct because ht_row_count exceeds 1024000 or other internal error, then merged bf can neither be constructed.
2. hash table is empty, partial bloom filter is null, it can be merged.